src/modules/esl-popup/core/esl-popup.ts

```diff
@@ -174,6 +170,21 @@ export class ESLPopup extends ESLToggleable {
     memoize.clear(this, '_offsetArrowRatio');
   }
 
+  /**
+   * Actions to execute after show popup.
```

Comment:
Comment sounds more correct like this: "Actions to execute after popup show." or "Actions to execute after show of popup"